src/components/plots/xyPlot.ts

```diff
@@ -123,31 +123,40 @@ module Plottable {
       return this;
     }
 
-    /**
-     * Sets the automatic domain adjustment over visible points for y scale.
-     *
-     * If autoAdjustment is true adjustment is immediately performend.
-     *
-     * @param {boolean} autoAdjustment The new value for the automatic adjustment domain for y scale.
-     * @returns {XYPlot} The calling XYPlot.
-     */
-    public automaticallyAdjustYScaleOverVisiblePoints(autoAdjustment: boolean): XYPlot<X, Y> {
-      this._autoAdjustYScaleDomain = autoAdjustment;
-      this._adjustYDomainOnChangeFromX();
-      return this;
-    }
-
-    /**
-     * Sets the automatic domain adjustment over visible points for x scale.
-     *
-     * If autoAdjustment is true adjustment is immediately performend.
-     *
-     * @param {boolean} autoAdjustment The new value for the automatic adjustment domain for x scale.
+    /** 
+     * Sets the automatic domain adjustment for visible points to operate against the X scale, Y scale, or neither.  
+     * 
+     * If 'x' or 'y' is specified the adjustment is immediately performed.
+     * 
+     * @param {string} scale Must be one of 'x', 'y', or 'none'.  
+     * 
+     * 'x' will adjust the x scale in relation to changes in the y domain.  
+     * 
+     * 'y' will adjust the y scale in relation to changes in the x domain.
+     * 
+     * 'none' means neither scale will change automatically.
+     * 
      * @returns {XYPlot} The calling XYPlot.
      */
-    public automaticallyAdjustXScaleOverVisiblePoints(autoAdjustment: boolean): XYPlot<X, Y>  {
-      this._autoAdjustXScaleDomain = autoAdjustment;
-      this._adjustXDomainOnChangeFromY();
+    public autorange(scale: string) {
+      switch (scale) {
+        case "x":
+          this._autoAdjustXScaleDomain = true;
+          this._autoAdjustYScaleDomain = false;
+          this._adjustXDomainOnChangeFromY();
+          break;
+        case "y":
+          this._autoAdjustXScaleDomain = false;
+          this._autoAdjustYScaleDomain = true;
+          this._adjustYDomainOnChangeFromX();
+          break;
+        case "none":
+          this._autoAdjustXScaleDomain = false;
+          this._autoAdjustYScaleDomain = false;
+          break;
+        default:
+          throw new Error("Invalid scale value '" + scale + "', must be 'x', 'y' or 'none'");
```

Comment:
Whatever decision we decide to make in the parameter above should probably be reflected in the error message here